projects/storefrontlib/shared/components/split-view/view/view.component.ts

```diff
@@ -65,6 +72,19 @@ export class ViewComponent implements OnInit, OnDestroy {
   @Output()
   hiddenChange = new EventEmitter();
 
+  @Input()
+  viewTitle?: string;
+
```

Comment:
Lets add TODOs here for cleanup. ie. `globalMessageService` is optional now but will be required next major. `featureConfigService` will be removed.

Reply:
done